tests/bluetooth/tester/nrf5340_hci_ipc.conf

```diff
@@ -1,5 +1,5 @@
 # Those have to be the same as in the controller (hci_ipc)
-CONFIG_BT_MAX_CONN=2
+CONFIG_BT_MAX_CONN=3
```

Comment:
Hmm, https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/bluetooth/hci_ipc/prj.conf sets `CONFIG_BT_MAX_CONN=16` and https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/bluetooth/hci_ipc/nrf5340_cpunet_iso-bt_ll_sw_split.conf sets `CONFIG_BT_MAX_CONN=2`, so this change to `CONFIG_BT_MAX_CONN=3` does not seem to conform to any of those, as per the comment.

Reply:
This is unrelated, because command which used to flash controller has nrf5340_hci_ipc.conf as second overlay config which will overwrite values from the previous one, hope that makes sense

west build -b nrf5340dk_nrf5340_cpunet -- -DOVERLAY_CONFIG='nrf5340_cpunet_iso-bt_ll_sw_split.conf;../../../tests/bluetooth/tester/nrf5340_hci_ipc_cpunet.conf'

Reply:
Hmm, are you sure? 

If so, then I would suggest to move the comment
```
# Those have to be the same as in the controller (hci_ipc)
```

to after `CONFIG_BT_MAX_CONN=3`, if it does not apply to that

Reply:
Added one more comment and moved old one as you suggested